crate/operator/change_backup.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from typing import Any
+
+import kopf
+
+from crate.operator.backup import update_backup_schedule
+from crate.operator.config import config
+from crate.operator.utils import crate
+from crate.operator.utils.kopf import StateBasedSubHandler
+from crate.operator.webhooks import (
+    WebhookBackupScheduleUpdatePayload,
+    WebhookEvent,
+    WebhookStatus,
+)
+
+
+class BackupScheduleUpdateSubHandler(StateBasedSubHandler):
+    @crate.on.error(error_handler=crate.send_update_failed_notification)
+    @crate.timeout(timeout=float(config.SCALING_TIMEOUT))
+    async def handle(  # type: ignore
+        self,
+        namespace: str,
+        name: str,
+        spec: kopf.Spec,
+        old: kopf.Body,
+        diff: kopf.Diff,
+        logger: logging.Logger,
+        **kwargs: Any,
+    ):
+
+        diff_found = None
+        for d in diff:
+            if d.operation == "change" and d.field == (
+                "spec",
+                "backups",
+                "aws",
+                "cron",
+            ):
+                diff_found = d
+
+        assert diff_found is not None
```

Comment:
If this happens, the handler would re-run forever (I think?). If we want to stop it completely, we need to raise a `kopf.PermanentError()`

Reply:
This doesn't make sense anymore because with the field handler it is guaranteed to only have 1 field